src/targets/rtr.rs

```diff
@@ -105,14 +114,17 @@ pub struct Tls {
     unit: Link,
     certificate: ConfigPath,
     key: ConfigPath,
+
+    #[serde(default = "Tcp::default_history_size")]
+    history_size: usize,
```

Comment:
Missing `#[serde(rename = "history-size")]`.

Reply:
Resolved by way of including `Tcp` instead.